logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/publicuser/GetAllKnownUsersUseCase.kt

```diff
@@ -0,0 +1,15 @@
+package com.wire.kalium.logic.feature.publicuser
+
+import com.wire.kalium.logic.data.publicuser.model.PublicUser
+import com.wire.kalium.logic.data.user.UserRepository
+import kotlinx.coroutines.flow.Flow
+
+interface GetAllKnownUsersUseCase {
+    suspend operator fun invoke(): Flow<List<PublicUser>>
+}
+
+class GetAllKnownUsersUseCaseImpl(private val userRepository: UserRepository) : GetAllKnownUsersUseCase {
```

Comment:
I'm lacking some context here, but if the purpose  of this UseCase is search, then some filtering is needed to filter out bots and leagelhold.

Reply:
Thanks Mohamed :) the purpose is indeed to get all known local contacts so we do not search anything just Get all local contacts, if this filtering is needed i think we also need to add this filtering into the searchknowncontacts use case because it is also just getting the users from the dao, maybe it is better to do it in the future for both of these use cases by adding a TODO so that we can keep going with the starting new conversation what do you think


P.s good night :p